src/figuresClasses.ts

```diff
@@ -1,11 +1,94 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
```

Comment:
```suggestion
export enum Color {
  Red = 'red',
  Blue = 'blue',
  Green = 'green',
}
export enum Shape {
  Triangle = 'triangle',
  Rectangle = 'rectangle',
  Circle = 'circle',
}

export interface Figure {
  color: Color;
  shape: Shape;
  getArea(): number;
}
```
give types more precisely